packages/web/package-lock.json

```diff
@@ -25,6 +25,7 @@
                 "react-responsive": "^9.0.0",
                 "react-router-dom": "^6.3.0",
                 "react-scripts": "5.0.1",
+                "react-toastify": "^9.1.3",
```

Comment:
What's up here? We already use react-hot-toast which I had linked to in my previous comment. Remove this. 


https://github.com/AUBGTheHUB/monolith/blob/207d674ce70a80912c7736a2fbe1abb5aa53475b/packages/web/src/App.js#L64

Reply:
https://github.com/AUBGTheHUB/monolith/blob/0d1948be7dd96bedabc139fe7e268ce1fe208505/packages/web/src/components/admin_page/s3_page/obj_card.jsx#L7